src/components/MainArea/GenresTabs/Genres.jsx

```diff
@@ -0,0 +1,52 @@
+import React from 'react';
+import styled, { css } from "styled-components";
+import PropTypes from 'prop-types'
+
+const UlAsTabs = styled.ul`
+    display: flex;
+    list-style-type: none;
+    font-family: 'Montserrat';
+    font-style: normal;
+    font-weight: 500;
+    font-size: 16px;
+    line-height: 20px;
+    
+    text-transform: uppercase;
+    
+    color: #FFFFFF;    
+`;
+
+const TabElement = styled.li`
+margin: 0 15px;
+cursor: pointer;
+
+.active {
+    background: pink;
+}
+`;
+
+
+function Genres(props) {
+    const elements= props.tabsList.map((item,i) => <TabElement key={i}>{item}</TabElement>)
+    return(
```

Comment:
What if tabList becomes undefined or null?
See what will happen because in reality you it could happen.

Reply:
rewrote this part:
1. added error boundary to genres
2. change genres from props to an array of objects to fix the issue with key from array